tests/parsing/LineParserTest.py

```diff
@@ -13,13 +14,24 @@ def test_empty_line(self):
         self.check_data_set('\n \n \n')
 
     def test_comment_parsing(self):
+        logger = logging.getLogger()
         self.check_data_set('# comment only$§\n',
                             output_comment='# comment only$§')
         self.check_data_set('   ; comment only  \n',
                             output_comment='; comment only')
         self.check_data_set('   ; \\comment only  \n',
                             output_comment='; comment only')
         self.check_data_set('#', output_comment='#')
+        with self.assertLogs(logger, 'WARNING') as cm:
+            self.check_data_set('##\n',
+                                output_comment='##')
+        self.assertEquals(cm.output, "The comment line does not have \
```

Comment:
I'll work on it but i don't think so after moving logger statement in the block, error will be removed.
I will pink you if i face any problem @nemaniarjun  :smile: